packages/material-ui-lab/src/Rating/Rating.js

```diff
@@ -107,6 +107,13 @@ export const styles = theme => ({
   iconActive: {
     transform: 'scale(1.2)',
   },
+  customIcons: {
+    display: 'flex',
+    alignItems: 'center',
+    '& > label > span > svg': {
+      margin: theme.spacing(2),
+    },
+  },
```

Comment:
Would it be better to leave this up to the developer, if they use non-standard icon sizes? This allows for a particular scenario which is not necessarily common.

Reply:
Good point. I'll tidy this up.